provisioner_test.go

```diff
@@ -119,3 +123,75 @@ func TestNewDisembedderProvisioner(t *testing.T) {
 	assert.NoError(t, err)
 	testProvisionerSuccessful(t, *p, "linux", "amd64", DefaultVersionAstilectron, DefaultVersionElectron)
 }
+
+func TestZipShouldRemove(t *testing.T) {
+	l := log.New(log.Writer(), log.Prefix(), log.Flags())
+
+	appName := "Test"
+	dataDir, _ := filepath.Abs("./testdata/tmp")
+	vendorDir := filepath.Join(dataDir, "vendor") // https://github.com/CarsonSlovoka/go-astilectron/blob/e7796e5/paths.go#L56
+	astilectronDownloadDst := filepath.Join(vendorDir, fmt.Sprintf("astilectron-v%s.zip", DefaultVersionAstilectron))
+	astilectronDir := filepath.Join(vendorDir, fmt.Sprintf("astilectron"))
+	electronDownloadDst := filepath.Join(vendorDir, fmt.Sprintf("electron-%s-%s-v%s.zip", runtime.GOOS, runtime.GOARCH, DefaultVersionElectron))
+	electronDir := filepath.Join(vendorDir, fmt.Sprintf("electron-%s-%s", runtime.GOOS, runtime.GOARCH))
+
+	t.Log("Make sure the test directory doesn't exist.")
+	if err := os.RemoveAll(dataDir); err != nil && !os.IsNotExist(err) {
+		t.FailNow()
+	}
+
+	t.Logf("Create a directory for test only:\n%s\n", dataDir)
```

Comment:
Remove all the debug logs below once you're done developping

Reply:
OK.